packages/terra-clinical-header/src/Header.jsx

```diff
@@ -2,6 +2,8 @@ import React, { useContext } from 'react';
 import PropTypes from 'prop-types';
 import classNames from 'classnames';
 import classNamesBind from 'classnames/bind';
+
+import HyperlinkButton from 'terra-hyperlink';
```

Comment:
Shouldn't we use Hyperlink instead of HyperlinkButton since that is the name of the component.  It makes troubleshooting more difficult if a different name is used.

Reply:
@cm9361 and all, the "Hyperlink Button" is technically made of a button but is programmatically understood as a link (required for screen reader users). The best practice is to always make links using a proper hyperlink (Terra Hyperlink). However, there are a few instances where a Hyperlink Button may be required instead of a link due to the need for code to use a button (purely technical implementation requirement because a link cannot be used). I cannot speak to whether this use case is 100% required to use a Hyperlink Button, but from an accessibility perspective, this would pass the accessibility requirements.

Reply:
@trandrew1023 I've reviewed the implementation and it looks good. I'll update the labels on the page to reflect that I've reviewed.

Reply:
We can also use Hyperlink instead and everything is still the same as long as onClick is passed to it. The reason for using HyperlinkButton however was because of the example on the Hyperlink doc for Hyperlink Button also uses HyperlinkButton

https://engineering.cerner.com/terra-ui/components/cerner-terra-core-docs/hyperlink/about

Reply:
I made this comment because the actual name of the component is Hyperlink.  In the examples provided, the component is imported as "Hyperlink".  As a developer, I would try to find a HyperlinkButton package if I saw this without reviewing the import statement.

Reply:
Yeah that's a good catch, the example for the hyperlink button with the `onClick` had that import, but good reminder to be more cautious especially after our discussion today on the .isRequired prop earlier not to take things at face value. Updated here https://github.com/cerner/terra-clinical/pull/923/commits/3cddf3d8242ff103129e2a6f8145bdf3ff3f970d

I think it would be good to update the example in since that can cause confusion for others. I can create that PR since it's quick instead of creating a Jira https://github.com/cerner/terra-core/blob/94880b5c38757f285c68ddb373c86ad0363af45b/packages/terra-core-docs/src/terra-dev-site/doc/hyperlink/example/DefaultHyperlinkButton.jsx#L3